app/controllers/Healthcheck.scala

```diff
@@ -1,12 +1,15 @@
 package controllers
 
-import actions.CommonActions.NoCacheAction
 import play.api.mvc._
+import services.TouchpointBackend
 
 object Healthcheck extends Controller {
 
-  def index = NoCacheAction {
-    Ok("""{"status":"OK"}""").withHeaders(CONTENT_TYPE -> "application/json")
+  def index = Action {
+    Cached(1) {
+      if(TouchpointBackend.Normal.zuoraService.products.size > 0) Ok("OK")
```

Comment:
I think so, the only think checking the healthcheck is the ELB which checks for a OK response. 
Good spot - will change to `nonEmpty`